lists/signature-types/README.md

```diff
@@ -0,0 +1,68 @@
+Signature Types
+===============
+
+This is a list of existing, reserved and tentative signature types
+for payloads signed using secp256k1.
+
+Signature Types
+---------------
+
+See the sections below for additional notes on prefix bytes marked
+Reserved or Tentative.
+
+| Prefix byte | Specs or Purpose |
+|-------------|------------------|
+| 0x00  | Reserved: indicates legacy (untyped) trancactions |
+| 0x01  | Reserved: [EIP-2930](https://eips.ethereum.org/EIPS/eip-2930) *(available in Berlin)* |
+| 0x02  | Reserved: [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559) *(available in London)* |
+| 0x03  | Tentative: prevents collision with [EIP-3074](https://eips.ethereum.org/EIPS/eip-3074) |
+| 0x18  | Reserved: prevents collision with [EIP-191](https://eips.ethereum.org/EIPS/eip-191) |
```

Comment:
Per https://eips.ethereum.org/EIPS/eip-191 this should be 0x19?

Reply:
Seems like it, yes!

Reply:
Yes! It should. :s

Reply:
Nice catch.

Reply:
```suggestion
| 0x19  | Reserved: prevents collision with [EIP-191](https://eips.ethereum.org/EIPS/eip-191) |
```